tools/ci/run_tc.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python
+
+import argparse
+import json
+import os
+import re
+import subprocess
+import sys
+
+
+root = os.path.abspath(
+    os.path.join(os.path.dirname(__file__),
+                 os.pardir,
+                 os.pardir))
+
+
+def run(cmd, return_stdout=False, **kwargs):
+    print(" ".join(cmd))
+    if return_stdout:
+        f = subprocess.check_output
+    else:
+        f = subprocess.check_call
+    return f(cmd, **kwargs)
+
+
+def get_parser():
+    p = argparse.ArgumentParser()
+    p.add_argument("job",
+                   help="Name of the job associated with the current event")
+    p.add_argument("script", help="Script to run for the job")
+    return p
+
+
+def get_extra_jobs(event):
+    body = None
+    jobs = set()
+    if "commits" in event:
+        body = event["commits"][0]["message"]
+    elif "pull_request" in event:
+        body = event["pull_request"]["body"]
+
+    if not body:
+        return jobs
+
+    regexp = re.compile("\s*tc-jobs:(.*)$")
+
+    for line in body.splitlines():
+        m = regexp.match(line)
+        if m:
+            items = m.group(1)
+            for item in items.split(","):
+                jobs.add(item.strip())
+            break
+    return jobs
+
+
+def set_variables(event):
+    # Set some variables that we use to get the commits on the current branch
+    ref_prefix = "refs/heads/"
+    pull_request = "false"
```

Comment:
`"false"` seems potentially more confusing than something else here: an empty string is falsy in most languages, unlike `"false"`, or just not setting the env variable?

Reply:
This is purely because that's what travis did.

Reply:
Avoiding confusion seems more valuable than mimicking TravisCI